app/components/on-this-page-links.hbs

```diff
@@ -0,0 +1,18 @@
+<div>
+  <div class="c-anchor-nav-holder">
+    <p>{{t "on-this-page"}}: </p>
+    <ul class="au-c-anchor-nav">
+      {{#each @items as |item|}}
+        <li>
+          <a
+            href={{concat "#" item.elementId}}
+            class="js-anchor"
+            onclick={{fn this.scrollToFragment item.elementId}}
```

Comment:
Does this take into account @ValenberghsSven 's fix to ensure the scroll works and doesn't add whitespace at the viewport bottom as it did in the past?